example/src/main/res/layout/main_activity.xml

```diff
@@ -8,6 +8,8 @@
     <com.quickbirdstudios.surveykit.survey.SurveyView
         android:id="@+id/survey_view"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        android:background="@color/color_primary_dark"
```

Comment:
```suggestion
```

Reply:
I'd like to keep the example app as it is for now. The other adjustments are fine!